src/diem/testing/suites/conftest.py

```diff
@@ -5,14 +5,18 @@
 from ... import testnet, jsonrpc
 from ..miniwallet import RestClient, AppConfig
 from .clients import Clients
-from .envs import target_url, is_self_check, should_test_debug_api
+from .envs import target_url, is_self_check, should_test_debug_api, dmw_offchain_host, dmw_offchain_port
 import pytest
 
 
 @pytest.fixture(scope="package")
 def target_client(diem_client: jsonrpc.Client) -> RestClient:
     if is_self_check():
-        conf = AppConfig(name="target-wallet", enable_debug_api=should_test_debug_api())
+        conf = AppConfig(
+            name="target-wallet",
+            enable_debug_api=should_test_debug_api(),
+            **diem_offchain_config(),
```

Comment:
I think this won't work, you are making the target and stub same host and port number.

Reply:
yeah I wasn't sure if it was needed for self check or not- I think it's fair to say it's not